src/env/set.c

```diff
@@ -0,0 +1,42 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*   tosh - 21 Shell                                      ::::::::            */
+/*                                                      :+:    :+:            */
+/*   By: aholster <aholster@student.codam.nl>          +:+                    */
+/*       ivan-tey <ivan-tey@student.codam.nl>         +#+                     */
+/*       nloomans <nloomans@student.codam.nl>        +#+                      */
+/*                                                 #+#    #+#                 */
+/*   License: GPLv3                                ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "private.h"
+#include <assert.h>
+
+int						env_set(t_env *const env,
+							char const *const key,
+							char const *const value)
+{
+	struct s_env_pair	*new;
+
+	assert(key != NULL || value != NULL);
```

Comment:
assert is le wrong

Reply:
oh also, is the sole assert so far, bit inconsistent

Reply:
I would just remove the `assert` here, because it's not done anywhere else.